code/datums/loadout/item_representation/_item_representation.dm

```diff
@@ -6,13 +6,16 @@
 /datum/item_representation
 	/// The type of the object represented, to allow us to create the object when needed
 	var/obj/item/item_type
+	/// Icon state of the item. This is so we can keep track of icon_state colors with the loadout vendor. Like PAS armor.
+	var/item_icon_state
```

Comment:
bad idea. If the icon state change in a future pr, you'll have no way of knowing this and the slot will be blank.